wordpress/user.py

```diff
@@ -0,0 +1,41 @@
+import wordpress
+from wordpress.client import APIClient
+
+
+class WPUser:
+    email = None
+    roles = None
+    user_id = None
+    
+    def __init__(self, client=APIClient(), logger=wordpress.logger):
+        self.logger = logger
+        self.roles = set()
+        self.client = client
+
+
+    def fetch(self, email):
+        response = self.client.get_request(f'users/?search={email}')
+        body = response.json()
+        if len(body):
+            user_id = body[0]['id']
+            response = self.client.post_request(f'users/{user_id}')
+            body = response.json()
+            email_profile = body['email']
+            if email == email_profile:
```

Comment:
isn't this check redundant? If at line 17 we fetched by the email what we know for sure is that emails match.

Reply:
Totally, not sure what i was thinking.